core/src/test/java/org/wildfly/extras/sunstone/tests/AbstractCloudProviderTest.java

```diff
@@ -274,15 +274,14 @@ private void testExecBuilder(Node node) throws IOException, InterruptedException
         assertThat(node.exec("cat", "/tmp/my-sudo-id.txt").getOutput(), containsString("root"));
 
         long sleepInSecs = 60;
-        ExecBuilder sleepCheckExec = ExecBuilder.fromCommand("pgrep", "sleep");
-        assertNotEquals("No running 'sleep' process should be found", 0, sleepCheckExec.exec(node).getExitCode());
+        assertEquals("No running 'sleep' process should be found", 1, node.exec("pgrep","sleep").getExitCode());
```

Comment:
IMHO in this case, `!= 0` is slightly better than `== 1`, but I don't really care :-)